MAFIA/Views/Controllers/DetailListPlayers/DetailListPlayersViewController.swift

```diff
@@ -84,17 +80,15 @@ class DetailListPlayersViewController: UIViewController {
         let emptyText = text ?? ""
         addPlayerAction?.isEnabled = !emptyText.isEmpty
     }
-
-    /*
-    // MARK: - Navigation
-
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        // Get the new view controller using segue.destinationViewController.
-        // Pass the selected object to the new view controller.
-    }
-    */
     
+    private func presentActionSheet(title: String?, message: String?) {
+        let alertActionSheet = UIAlertController(title: title, message: message, preferredStyle: .actionSheet)
+        let okayAction = UIAlertAction(title: "OKAY".localized(), style: .default, handler: nil)
+        
```

Comment:
What about making this function in an extension of UIViewcontroller, so any controller can use it.